docs/anomaly-detection/presets/README.md

```diff
@@ -0,0 +1,42 @@
+---
```

Comment:
why do you need a separate folder? why not just a big long document?

Reply:
It was a single doc previously. Rolled back to 1 file version.